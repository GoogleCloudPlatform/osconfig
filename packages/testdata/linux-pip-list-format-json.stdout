[{"name": "attrs", "version": "19.3.0"}, {"name": "Automat", "version": "0.8.0"}, {"name": "blinker", "version": "1.4"}, {"name": "certifi", "version": "2019.11.28"}, {"name": "chardet", "version": "3.0.4"}, {"name": "Click", "version": "7.0"}, {"name": "cloud-init", "version": "24.3.1"}, {"name": "colorama", "version": "0.4.3"}, {"name": "command-not-found", "version": "0.3"}, {"name": "configobj", "version": "5.0.6"}, {"name": "constantly", "version": "15.1.0"}, {"name": "cryptography", "version": "2.8"}, {"name": "dbus-python", "version": "1.2.16"}, {"name": "distro", "version": "1.4.0"}, {"name": "distro-info", "version": "0.23+ubuntu1.1"}, {"name": "entrypoints", "version": "0.3"}, {"name": "httplib2", "version": "0.14.0"}, {"name": "hyperlink", "version": "19.0.0"}, {"name": "idna", "version": "2.8"}, {"name": "importlib-metadata", "version": "1.5.0"}, {"name": "incremental", "version": "16.10.1"}, {"name": "Jinja2", "version": "2.10.1"}, {"name": "jsonpatch", "version": "1.22"}, {"name": "jsonpointer", "version": "2.0"}, {"name": "jsonschema", "version": "3.2.0"}, {"name": "keyring", "version": "18.0.1"}, {"name": "language-selector", "version": "0.1"}, {"name": "launchpadlib", "version": "1.10.13"}, {"name": "lazr.restfulclient", "version": "0.14.2"}, {"name": "lazr.uri", "version": "1.0.3"}, {"name": "MarkupSafe", "version": "1.1.0"}, {"name": "more-itertools", "version": "4.2.0"}, {"name": "netifaces", "version": "0.10.4"}, {"name": "oauthlib", "version": "3.1.0"}, {"name": "pexpect", "version": "4.6.0"}, {"name": "pip", "version": "20.0.2"}, {"name": "pyasn1", "version": "0.4.2"}, {"name": "pyasn1-modules", "version": "0.2.1"}, {"name": "PyGObject", "version": "3.36.0"}, {"name": "PyHamcrest", "version": "1.9.0"}, {"name": "PyJWT", "version": "1.7.1"}, {"name": "pymacaroons", "version": "0.13.0"}, {"name": "PyNaCl", "version": "1.3.0"}, {"name": "pyOpenSSL", "version": "19.0.0"}, {"name": "pyrsistent", "version": "0.15.5"}, {"name": "pyserial", "version": "3.4"}, {"name": "python-apt", "version": "2.0.1+ubuntu0.20.4.1"}, {"name": "python-debian", "version": "0.1.36+ubuntu1.1"}, {"name": "PyYAML", "version": "5.3.1"}, {"name": "requests", "version": "2.22.0"}, {"name": "requests-unixsocket", "version": "0.2.0"}, {"name": "SecretStorage", "version": "2.3.1"}, {"name": "service-identity", "version": "18.1.0"}, {"name": "setuptools", "version": "45.2.0"}, {"name": "simplejson", "version": "3.16.0"}, {"name": "six", "version": "1.14.0"}, {"name": "sos", "version": "4.5.6"}, {"name": "ssh-import-id", "version": "5.10"}, {"name": "systemd-python", "version": "234"}, {"name": "Twisted", "version": "18.9.0"}, {"name": "ubuntu-pro-client", "version": "8001"}, {"name": "ufw", "version": "0.36"}, {"name": "unattended-upgrades", "version": "0.1"}, {"name": "urllib3", "version": "1.25.8"}, {"name": "wadllib", "version": "1.3.3"}, {"name": "wheel", "version": "0.34.2"}, {"name": "zipp", "version": "1.0.0"}, {"name": "zope.interface", "version": "4.7.1"}]
