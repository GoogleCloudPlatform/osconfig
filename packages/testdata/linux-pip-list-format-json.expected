[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "attrs",
        Arch:    "all",
        RawArch: "",
        Version: "19.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "Automat",
        Arch:    "all",
        RawArch: "",
        Version: "0.8.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "blinker",
        Arch:    "all",
        RawArch: "",
        Version: "1.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "certifi",
        Arch:    "all",
        RawArch: "",
        Version: "2019.11.28",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "chardet",
        Arch:    "all",
        RawArch: "",
        Version: "3.0.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "Click",
        Arch:    "all",
        RawArch: "",
        Version: "7.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cloud-init",
        Arch:    "all",
        RawArch: "",
        Version: "24.3.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "colorama",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "command-not-found",
        Arch:    "all",
        RawArch: "",
        Version: "0.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "configobj",
        Arch:    "all",
        RawArch: "",
        Version: "5.0.6",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "constantly",
        Arch:    "all",
        RawArch: "",
        Version: "15.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cryptography",
        Arch:    "all",
        RawArch: "",
        Version: "2.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "dbus-python",
        Arch:    "all",
        RawArch: "",
        Version: "1.2.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "distro",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "distro-info",
        Arch:    "all",
        RawArch: "",
        Version: "0.23+ubuntu1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "entrypoints",
        Arch:    "all",
        RawArch: "",
        Version: "0.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "httplib2",
        Arch:    "all",
        RawArch: "",
        Version: "0.14.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "hyperlink",
        Arch:    "all",
        RawArch: "",
        Version: "19.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "idna",
        Arch:    "all",
        RawArch: "",
        Version: "2.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "importlib-metadata",
        Arch:    "all",
        RawArch: "",
        Version: "1.5.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "incremental",
        Arch:    "all",
        RawArch: "",
        Version: "16.10.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "Jinja2",
        Arch:    "all",
        RawArch: "",
        Version: "2.10.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "jsonpatch",
        Arch:    "all",
        RawArch: "",
        Version: "1.22",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "jsonpointer",
        Arch:    "all",
        RawArch: "",
        Version: "2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "jsonschema",
        Arch:    "all",
        RawArch: "",
        Version: "3.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "keyring",
        Arch:    "all",
        RawArch: "",
        Version: "18.0.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "language-selector",
        Arch:    "all",
        RawArch: "",
        Version: "0.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "launchpadlib",
        Arch:    "all",
        RawArch: "",
        Version: "1.10.13",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "lazr.restfulclient",
        Arch:    "all",
        RawArch: "",
        Version: "0.14.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "lazr.uri",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "MarkupSafe",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "more-itertools",
        Arch:    "all",
        RawArch: "",
        Version: "4.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "netifaces",
        Arch:    "all",
        RawArch: "",
        Version: "0.10.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "oauthlib",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pexpect",
        Arch:    "all",
        RawArch: "",
        Version: "4.6.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pip",
        Arch:    "all",
        RawArch: "",
        Version: "20.0.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pyasn1",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pyasn1-modules",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "PyGObject",
        Arch:    "all",
        RawArch: "",
        Version: "3.36.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "PyHamcrest",
        Arch:    "all",
        RawArch: "",
        Version: "1.9.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "PyJWT",
        Arch:    "all",
        RawArch: "",
        Version: "1.7.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pymacaroons",
        Arch:    "all",
        RawArch: "",
        Version: "0.13.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "PyNaCl",
        Arch:    "all",
        RawArch: "",
        Version: "1.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pyOpenSSL",
        Arch:    "all",
        RawArch: "",
        Version: "19.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pyrsistent",
        Arch:    "all",
        RawArch: "",
        Version: "0.15.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pyserial",
        Arch:    "all",
        RawArch: "",
        Version: "3.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "python-apt",
        Arch:    "all",
        RawArch: "",
        Version: "2.0.1+ubuntu0.20.4.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "python-debian",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.36+ubuntu1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "PyYAML",
        Arch:    "all",
        RawArch: "",
        Version: "5.3.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "requests",
        Arch:    "all",
        RawArch: "",
        Version: "2.22.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "requests-unixsocket",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "SecretStorage",
        Arch:    "all",
        RawArch: "",
        Version: "2.3.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "service-identity",
        Arch:    "all",
        RawArch: "",
        Version: "18.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "setuptools",
        Arch:    "all",
        RawArch: "",
        Version: "45.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "simplejson",
        Arch:    "all",
        RawArch: "",
        Version: "3.16.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "six",
        Arch:    "all",
        RawArch: "",
        Version: "1.14.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "sos",
        Arch:    "all",
        RawArch: "",
        Version: "4.5.6",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ssh-import-id",
        Arch:    "all",
        RawArch: "",
        Version: "5.10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd-python",
        Arch:    "all",
        RawArch: "",
        Version: "234",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "Twisted",
        Arch:    "all",
        RawArch: "",
        Version: "18.9.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ubuntu-pro-client",
        Arch:    "all",
        RawArch: "",
        Version: "8001",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ufw",
        Arch:    "all",
        RawArch: "",
        Version: "0.36",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "unattended-upgrades",
        Arch:    "all",
        RawArch: "",
        Version: "0.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "urllib3",
        Arch:    "all",
        RawArch: "",
        Version: "1.25.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "wadllib",
        Arch:    "all",
        RawArch: "",
        Version: "1.3.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "wheel",
        Arch:    "all",
        RawArch: "",
        Version: "0.34.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "zipp",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "zope.interface",
        Arch:    "all",
        RawArch: "",
        Version: "4.7.1",
        Source:  packages.Source{},
    },
}